msdsl/model.py

```diff
@@ -513,15 +519,19 @@ def make_uint_table(self, vals, width=None, name=None, dir=None):
 
     # function creation
 
-    def make_function(self, func, domain, name=None, dir=None, **kwargs):
+    def make_function(self, func, domain, name=None, dir=None, real_type=None,
```

Comment:
there is usually no need to define **real_type**, since it defaults to the value from the class instantiation